### PR TITLE
 Fix:Issue #13965 resolved. <code> block in the UI must not show now when there is an error installing any extension. 

### PR DESCRIPTION
**Pull request by doesnots**
## References

Fixes https://github.com/jupyterlab/jupyterlab/issues/13965

### CODE REVIEW

**Review by andrii-i — APPROVED** [review]
This looks good, follows suggestions, and fixes #13965. Thank you, @doesnots.

@doesnots could you please also fill out the template in this PRs header so that people looking at it in the future would be able to quickly see what this PR is about?

**Review by andrii-i — CHANGES_REQUESTED** [review]
There are 2 tests currently failing: 
* [Depreciated har-validator@5.1.5 library warning ](https://github.com/jupyterlab/jupyterlab/actions/runs/4281879501/jobs/7455444442) which does not seem to be related to this change
* [Linter fails](https://github.com/jupyterlab/jupyterlab/actions/runs/4281879501/jobs/7455446292). @doesnots could you please make sure you ran pre-commit hooks on this change? They should run linter and correct all linting problems

File: packages/extensionmanager/src/dialog.tsx [hunk 1]
```diff
@@ -19,9 +19,7 @@ export function reportInstallError(
   translator = translator || nullTranslator;
   const trans = translator.load('jupyterlab');
   const entries = [];
-  entries.push(
-    <p>{trans.__('An error occurred installing <code>%1</code>.', name)}</p>
-  );
+  entries.push(<p>{trans.__(`An error occurred installing "${name}."`)}</p>);
```
Comment (jtpio) [inline packages/extensionmanager/src/dialog.tsx:22]: Looks like the dot could be moved:

```suggestion
  entries.push(<p>{trans.__(`An error occurred installing "${name}".`)}</p>);
```

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**